generate.ts

```diff
@@ -81,7 +85,7 @@ function reportErrorsAndExit(errorMessages) {
 
     // Else, report all errors and exit the program
     console.log("Syntax: node generate.js\n" +
-        "\t\t<targetName>=<targetOutputPath>\n" +
+        "\t\t(<targetName>=<targetOutputPath>|-destPath <destFolderPath>)\n" +
```

Comment:
>-destPath <destFolderPath> [](start = 46, length = 26)

When using -destPath argument, can we consider using an additional parameter -getConfigPath=<path to getConfig.json file>? Sure, in most cases it will point to a file located at the root of destPath, but.. not necessarily, for advanced scenarios? It would be cool to keep door open for additional flexibility

Reply:
That's an interesting idea. I think we could add it later if we so decided.
I don't think it needs to come in with this pass of this feature.